### ISSUE TITLE
Swagger aggregation

### ISSUE THREAD

**Issue by mteodori** [Activiti/Activiti#2304]

**Comment by salaboy** [comment]
@mteodori please feel free to re-assign to whoever will work on this. I am adding it to RC1, because I think that we will need this in a stable release

**Comment by Jary1** [comment]
mute the thread



At 2018-12-23 20:12:09, "salaboy" <notifications@github.com> wrote:


@mteodori please feel free to re-assign to whoever will work on this. I am adding it to RC1, because I think that we will need this in a stable release

**Comment by salaboy** [comment]
@mteodori yes it is.. we are releasing the gateway in each major release  (https://github.com/activiti/activiti-cloud-gateway) so you will find the tags in there. It is also part of our infrastructure, so yes.. it is deployed by the full example HELM chart as part of the sub Infrastructure HELM chart.. If we add the aggregator in the gateway, we will see the benefits straight away.

**Comment by salaboy** [comment]
@mteodori did you found a way to aggregate swagger files?

**Comment by salaboy** [comment]
After syncing with the team and @igdianov we all agree that not having a single swagger json file to interactively call the services. Having postman collections is not sustainable and we are hurting users that want to start testing the services.
